docs/rules.md

```diff
@@ -363,6 +364,18 @@ These are considered invalid:
     end
 
 
+### NoInterpolationOnlyStrings
+
+A rule that disallows strings which are entirely the result of an
+interpolation.
+
+Good:
+    output = "Dogma.Rule.NoInterpolationOnlyStrings"
```

Comment:
For the string that is the argument to the moduledoc, not for the string in the example. Currently the #{} is being interpolated into the moduledoc's string, and thus doesn't show in the rendered example.